src/runtime/auto-prompt-manager-test.js

```diff
@@ -3696,7 +3696,7 @@ describes.realWin('AutoPromptManager', (env) => {
       expect(autoPromptManager.isInDevMode_).to.be.false;
     });
 
-    describe('when dismissibility filter experiment enabled', () => {
```

Comment:
Seems like the describe should be removed and the tests unindented if it's no longer applicable.